packages/icondusk-e2e/package.py

```diff
@@ -18,6 +18,7 @@ class IconduskE2e(CMakePackage):
     maintainers = ['cosunae']
 
     version('master', branch='master')
+    version('0.1-dev', branch='master')
```

Comment:
@Stagno do you think this is a good version specifier? Not sure if we want to use a _symbolic_ one. I generally think a numeric (`0.1` or similar) is fine.

Reply:
```suggestion
    version('0.1-dev', tag='0.1-dev')
```
I think the numeric version is super fine.
Although i previously suggested that the main `icondusk-e2e` plan should build this version (and not master), I now think this is not necessary anymore (as also master version uses `dusk@icondusk-e2e` by default) ... we can make that plan just plainly build `master` version and thus we wouldn't need this version anymore...
It's anyways worth keeping it, and making it point to a `0.1-dev` tag instead of branch master, because, as we proceed with the ICON integration, we will depend on this repo to fetch the dusk stencils to use and certainly it's also a good idea to use a fixed commit there.

Reply:
Or maybe it's also a good idea to replicate the same "rolling tag" approach for the future ICON fork -> icondusk-e2e dependency... so having a `ICON-DSL` version (and tag) on icondusk-e2e... But i think for now `0.1-dev` is just fine.